pyvista/core/filters/poly_data.py

```diff
@@ -2755,6 +2755,113 @@ def extrude_rotate(
             return self
         return output
 
+    def extrude_trim(
+        self,
+        direction,
+        trim_surface,
+        extrusion="boundary_edges",
+        capping="intersection",
+        inplace=False,
+        progress_bar=False,
+    ):
+        """Extrude polygonal data trimmed by a surface.
+
+        This generates polygonal data on output.
+
+        The input dataset is swept along a specified direction forming a
+        "skirt" from the boundary edges 2D primitives (i.e., edges used
```

Comment:
Is this description also accurate when `extrusion="all_edges"`.  I would assume you get a filled unstructured grid or structured grid?

Reply:
The other extrusion filters return polydata with polystrips at worst, if memory serves. Have to check here to be sure, of course.

Reply:
Return is always a `PolyData`.